.github/workflows/test-playwright-full.yml

```diff
@@ -115,13 +115,17 @@ jobs:
         working-directory: site/gatsby-site
 
       - name: Run playwright tests
-        run: npx playwright test playwright/e2e-full/ --shard=${{ matrix.shardIndex }}/${{ matrix.shardTotal }}
+        run: npm run test:e2e:ci
```

Comment:
This is so we can tweak the playwright's command without having to merge the GitHub workflow.